### PR TITLE
BLOCKCHAIN-407 - FE staking add additional information

### PR DESCRIPTION
**Pull request by SzymonZiebaNeti**

### CODE REVIEW

File: src/components/Modals/AgreeDisagreeModal.js [hunk 1]
```diff
@@ -11,17 +11,19 @@ function AgreeDisagreeModal({
...
+  buttonRight,
 }) {
   return (
     <div className={classNames(styles.agreeDisagree, style)}>
       <h3>{text}</h3>
       {children || <span>This operation costs 100 LLD.</span>}
       <div className={styles.buttons}>
         <Button type="button" onClick={onDisagree} medium>
-          Cancel
+          {buttonLeft || 'Cancel'}
```
Comment (DorianSternVukotic) [inline src/components/Modals/AgreeDisagreeModal.js:23]: This seems unnecessary? Was it needed or just a preference?

File: src/utils/staking.js [hunk 2]
```diff
@@ -70,3 +73,114 @@ export const stakingInfoToProgress = (stakingInfo, progress) => {
...
+
+const DEFAULT_PARAMS = {
+  auctionAdjust: 0,
+  auctionMax: 0,
+  // 5% for falloff, as per the defaults, see
+  falloff: 0.05,
+  // 10% max, 0.25% min, see
+  maxInflation: 0.1,
+  minInflation: 0.025,
+  stakeTarget: 0.5,
```
[the remark this answers is not in the document]
Comment (kacperzuk-neti) [reply src/utils/staking.js:85]: Specifically, here: https://github.com/liberland/liberland_substrate/blob/develop/substrate/bin/node/runtime/src/lib.rs#L588-L593

File: src/utils/staking.js [hunk 3]
```diff
@@ -70,3 +73,114 @@ export const stakingInfoToProgress = (stakingInfo, progress) => {
         ),
     }));
 };
+
+const DEFAULT_PARAMS = {
+  auctionAdjust: 0,
+  auctionMax: 0,
```
Comment (kacperzuk-neti) [inline src/utils/staking.js:79]: Let's drop all auction-related logic. It's kusama-specific stuff.

File: src/components/Modals/AgreeDisagreeModal.js (hunk 1 above)
Comment (SzymonZiebaNeti) [reply src/components/Modals/AgreeDisagreeModal.js:23]: I used it in popup, when leaving page without saving. Made this popup more usable.

```
 <AgreeDisagreeModal
          text="You have unsaved changes. What would you like to do with them?"
          buttonLeft="Discard"
          buttonRight="Update"
          style={stylesModal.getCitizenshipModal}
          onDisagree={handleDiscardChanges}
          onAgree={() => updateNominations(selectedValidatorsAsTargets)}
        >
```

**Pull request merged**